### ISSUE TITLE
No Gyroscope/motion support

### ISSUE THREAD

**Issue by Peppernose145** [SachinVin/citra_android#236]
Citra for Android does not utilize the native gyroscope sensors of any device for emulation. This makes games such as WarioWare Gold partially unplayable. I think it should be very easy for someone to implement this feature since Android has a library for accessing sensors.

**Comment by VKarazhekov** [comment]
The motion controls can be emulated via the touch screen. Is your idea to make the 3DS gyroscope controls tied with the actual gyroscope of the mobile handset?

**Comment by Peppernose145** [comment]
Yes. My idea was to use the real gyroscope hardware on any given device to
emulate the 3DS' gyroscope.
